dao/src/main/java/org/thingsboard/server/dao/alarm/BaseAlarmService.java

```diff
@@ -28,26 +28,16 @@
 import org.springframework.util.StringUtils;
 import org.thingsboard.common.util.ThingsBoardThreadFactory;
 import org.thingsboard.server.common.data.Tenant;
-import org.thingsboard.server.common.data.alarm.Alarm;
-import org.thingsboard.server.common.data.alarm.AlarmInfo;
-import org.thingsboard.server.common.data.alarm.AlarmQuery;
-import org.thingsboard.server.common.data.alarm.AlarmSearchStatus;
-import org.thingsboard.server.common.data.alarm.AlarmSeverity;
-import org.thingsboard.server.common.data.alarm.AlarmStatus;
+import org.thingsboard.server.common.data.alarm.*;
```

Comment:
We don't do import optimizations. See https://peterdev.pl/2019/10/18/why-you-should-avoid-star-imports-in-intellij-idea/